python/sdk/src/repl.py

```diff
@@ -0,0 +1,82 @@
+from collections import deque
+
+
+import httpx
+from httpx_ws import WebSocketSession, connect_ws
+
+
```

Comment:
nit: is this spacing standard? can you run `uvx ruff format` in the base here just to clean up anything that deviates from ruff?

Reply:
I used black, reformatting with ruff!